ui/src/overridden/backoffice/Items/ItemCirculationShelf.js

```diff
@@ -22,6 +23,15 @@ export const ItemCirculationShelf = ({ metadata }) => {
         </Button>
       </>
     </Grid.Column>
+  ) : (
+    <Grid.Column width={6}>
+      <Message warning icon style={{ justifyContent: "center" }}>
```

Comment:
please remember not to place inline styling, to be moved to respective less stylesheet `.overrides `